core/Updater.php

```diff
@@ -279,6 +279,15 @@ public function update($componentName)
 
         $this->executeListenerHook('onComponentUpdateFinished', array($componentName, $updatedVersion, $warningMessages));
 
+        /**
+         * Event triggered after a component has been updated.
+         *
+         * @param string $componentName 'core', or plugin name
+         * @param string $updatedVersion version updated to
+         * @param array  $warningMessages warnings occurred during update
+         */
+        Piwik::postEvent('Updater.componentUpdated', array($componentName, $updatedVersion, $warningMessages));
```

Comment:
I'd be tempted to remove the observer completely and to replace it. It's not really "the Piwik way" and seems bit unneeded/over-engineered. However, there might be some benefits of having it so let's just leave it. 

I was then wondering if we could use the observer/hook instead of the event but I think it's better to have the event as this is API 👍